docs/getting-started/my-activity/README.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_navigation:
+  title: My Activity
+  priority: 200
+description: My Activity in OpenProject - your latest actions
+keywords: my activity, personal changes, dashboard
+---
+
+# My Activity
+
+**My Activity** page gives you an overview of all your latest actions and projects you are involved in. 
```

Comment:
I would either say:

- "**My activity** gives you..." (without "page") or
- "The **My activity** page gives you" (with "page" but also with "The" at the start")

Reply:
going for the first option.